xml/src/main/resources/ehcache-core.xsd

```diff
@@ -221,13 +221,13 @@
               </xs:complexType>
             </xs:element>
           </xs:sequence>
-          <xs:attribute type="xs:positiveInteger" name="concurrency" default="1" use="optional"/>
-          <xs:attribute type="xs:positiveInteger" name="size" default="2147483647" use="optional"/>
-          <xs:attribute type="xs:boolean" name="coalesce" default="false" use="optional"/>
-          <xs:attribute name="ratelimitpersecond" type="xs:positiveInteger" default="2147483647" use="optional"/>
-          <xs:attribute name="minWriteDelay" type="xs:nonNegativeInteger" default="0" use="optional"/>
-          <xs:attribute name="maxWriteDelay" type="xs:positiveInteger" default="2147483647" use="optional"/>
-          <xs:attribute name="batchsize" type="xs:positiveInteger" default="1" use="optional"/>
+          <xs:attribute name="concurrency" type="xs:positiveInteger" default="1" use="optional"/>
+          <xs:attribute name="size" type="xs:positiveInteger" default="2147483647" use="optional"/>
+          <xs:attribute name="coalesce" type="xs:boolean" default="false" use="optional"/>
```

Comment:
I could put it in a separate commit - main reason is readability, I actually missed these when I did work on #534